Robot2018/src/org/usfirst/frc/team199/Robot2018/commands/ShiftDrive.java

```diff
@@ -0,0 +1,52 @@
+package org.usfirst.frc.team199.robot.commands;
+
+import org.usfirst.frc.team199.robot.Robot;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ *
+ */
```

Comment:
can you make two separate commands/buttons for high and low gear, please? Just based on offseason the drivers really preferred this.